zadanie2/css/components/post/_post.scss

```diff
@@ -0,0 +1,51 @@
+.post {
+    margin-bottom: 20px;
+    background-color: $color-white;
+    border: 1px solid $color-border;
+    border-radius: 7px;
+    &:last-child {
+        margin-bottom: 0;
+    }
+    &__header {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 15px;
+    }
+    &__details {
+        padding: 15px;
+    }
+    &__actions {
+        display: flex;
+        align-items: center;
+        margin-bottom: 15px;
+        &-item {
+            margin-right: 20px;
+            img {
+                display: block;
+            }
+            &:last-child {
+                margin-right: 0;
+            }
+        }
+    }
+    &__comments {
```

Comment:
jesli nigdzie w kodzie nie wystepuje klasa "post-comments" i nie masz dla niej osobno styli, to nie ma potrzeby tego rozbijac

moglas to zapisac w 1 linii po prostu jako "&__comments-comment" :)